blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -174,8 +174,13 @@
     // 3 days - this matches the TTL for our metrics_full table, we don't accept anything older than the TTL.
     BEFORE_CURRENT_COLLECTIONTIME_MS("259200000"),
     // 10 minutes
-    AFTER_CURRENT_COLLECTIONTIME_MS("600000");
+    AFTER_CURRENT_COLLECTIONTIME_MS("600000"),
 
+    // Cross-Origin Resource Sharing
+    CORS_ALLOWED_ORIGINS(""),
```

Comment:
Is empty String a valid value for this header? This w3C page doesn't say it's valid:
https://www.w3.org/TR/2008/WD-access-control-20080912/#access-control-allow-origin